portfolio/src/main/java/com/google/sps/servlets/MarkerServlet.java

```diff
@@ -5,8 +5,9 @@
 import com.google.appengine.api.datastore.Entity;
 import com.google.appengine.api.datastore.PreparedQuery;
 import com.google.appengine.api.datastore.Query;
-import com.google.gson.Gson;
+import com.google.sps.data.Location;
 import com.google.sps.data.Marker;
+import com.google.gson.Gson;
```

Comment:
It looks like you moved this import out of order. It should be before `import com.google.sps.data.Location` to be in alphabetical order